soil/templates/soil/partials/dl_status.html

```diff
@@ -1,4 +1,5 @@
 {% load i18n %}
+{% load hq_shared_tags %}
```

Comment:
hmm good point. i don't have enough context on django-soil to know where it should be located. happy to move it to ex-submodule (probably in another PR). currently the only thing depending on it is the toggle. so when i remove the toggle, i can remove this `hq_shared_tags` and it'll be independent of HQ again